Sources/Quick/Configuration/QuickConfiguration.swift

```diff
@@ -37,6 +37,7 @@ extension QuickConfiguration {
             guard
                 let subclass = classes[Int(index)],
                 let superclass = class_getSuperclass(subclass),
+                type(of: superclass) == QuickConfiguration.Type.self,
```

Comment:
Unfortunately this doesn't work for cases of indirect subclasses:

```swift
class BaseConfig: QuickConfiguration { ... }
class FooConfig: BaseConfig { ... }
```

That is the reason why we use `superclass is QuickConfiguration.Type` here.

ref: #975